app/portainer/components/option-panel/option-panel.html

```diff
@@ -0,0 +1,15 @@
+<ng-form class="form-horizontal" name="$ctrl.optionForm">
+  <div class="form-group">
+    <div class="col-sm-12 form-section-title" style="margin-top: 10px; margin-left: 15px; width: 98%"> Options </div>
+
+    <!-- Prune service -->
+    <div class="col-sm-12">
+      <label class="control-label text-left">
+        Prune services
+        <portainer-tooltip position="top" message="Prune service that are no longer referenced."></portainer-tooltip>
```

Comment:
```suggestion
        <portainer-tooltip position="top" message="Prune services that are no longer referenced."></portainer-tooltip>
```

Reply:
Good catch, thanks @itsconquest Will update it